src/components/PokeImage/PokeBallImage.js

```diff
@@ -0,0 +1,23 @@
+import "./PokeImage.scss"
+import { registerHtml, useEffect, useGlobalState } from "tram-one"
+import Pokedex from 'pokedex-promise-v2'
+const PokeAPI = new Pokedex()
+
+const html = registerHtml()
+
+export default () => {
+  const [pokeBall, setPokeBall] = useGlobalState('pokeBall', null)
+
+  useEffect(async () => {
+    const fetchedPokeBall = await PokeAPI.getItemByName('poke-ball')
+    setPokeBall(fetchedPokeBall)
+  })
```

Comment:
not super passionate about it, but maybe we should just hard-code this url, rather than making a request every time?